cl/users/views.py

```diff
@@ -65,15 +66,29 @@ def view_alerts(request: HttpRequest) -> HttpResponse:
         # default to 'o' because if there's no 'type' param in the search UI,
         # that's an opinion search.
         a.type = QueryDict(a.query).get("type", SEARCH_TYPES.OPINION)
+    return render(
```

Comment:
It's probably worth renaming this function to `view_search_alerts`, no?

Reply:
yeah, changed!